docs/ops/convolution/BinaryConvolution_1.md

```diff
@@ -4,20 +4,55 @@
 
 **Attributes**:
 
-The operation has the same attributes as a regular *Convolution* layer and several unique attributes that are listed below:
+* *strides*
+
+  * **Description**: *strides* is a distance (in pixels) to slide the filter on the feature map over the `(z, y, x)` axes for 3D convolutions and `(y, x)` axes for 2D convolutions. For example, *strides* equal `4,2,1` means sliding the filter 4 pixel at a time over depth dimension, 2 over height dimension and 1 over width dimension.
+  * **Range of values**: integer values starting from 0
+  * **Type**: int[]
+  * **Default value**: None
+  * **Required**: *yes*
+
+* *pads_begin*
+
+  * **Description**: *pads_begin* is a number of pixels to add to the beginning along each axis. For example, *pads_begin* equal `1,2` means adding 1 pixel to the top of the input and 2 to the left of the input.
+  * **Range of values**: integer values starting from 0
+  * **Type**: int[]
+  * **Default value**: None
+  * **Required**: *yes*
+  * **Note**: the attribute is ignored when *auto_pad* attribute is specified.
+
+* *pads_end*
+
+  * **Description**: *pads_end* is a number of pixels to add to the ending along each axis. For example, *pads_end* equal `1,2` means adding 1 pixel to the bottom of the input and 2 to the right of the input.
+  * **Range of values**: integer values starting from 0
+  * **Type**: int[]
+  * **Default value**: None
+  * **Required**: *yes*
+  * **Note**: the attribute is ignored when *auto_pad* attribute is specified.
+
+* *dilations*
+
+  * **Description**: *dilations* denotes the distance in width and height between elements (weights) in the filter. For example, *dilation* equal `1,1` means that all the elements in the filter are neighbors, so it is the same as for the usual convolution. *dilation* equal `2,2` means that all the elements in the filter are matched not to adjacent elements in the input matrix, but to those that are adjacent with distance 1.
+  * **Range of values**: integer value starting from 0
+  * **Type**: int[]
+  * **Default value**: None
+  * **Required**: *yes*
 
 * *mode*
 
-  * **Description**: *mode* defines how input tensor 0/1 values and weights 0/1 are interpreted as real numbers and how the result is computed.
+  * **Description**: *mode* defines how input tensor `0/1` values and weights `0/1` are interpreted as real numbers and how the result is computed.
   * **Range of values**:
     * *xnor-popcount*
   * **Type**: `string`
   * **Default value**: None
   * **Required**: *yes*
+  *  **Note**: value `0` in inputs is interpreted as `-1`, value `1` as `1`
```

Comment:
Based on [this](https://sushscience.wordpress.com/2017/10/01/understanding-binary-neural-networks/) article I plan to use bellow formula to do the computations. I would add it to the spec, but I'm not sure if it's correct at this point.

`XNOR(A, B) = 00010
Popcount P of 00010 = 1
Result = 2*P - N, where N is the total number of bits

In this case 2*1 - 5 = -3`